udata_hydra_csvapi/utils.py

```diff
@@ -0,0 +1,41 @@
+from slugify import slugify
+
+
+def build_sql_query_string(request_arg: list, page_size: int, offset: int = 0) -> str:
+    sql_query = []
+    for arg in request_arg:
+        argument, value = arg.split('=')
+        if '__' in argument:
+            column, comparator = argument.split('__')
+            normalized_column = slugify(column, separator='_', allow_unicode=True)
```

Comment:
Since it's the user's role to encode the query accordingly, we don't need to slugify anymore, do we?